pkg/rtc/room.go

```diff
@@ -257,6 +260,10 @@ func (r *Room) RemoveParticipant(identity string) {
 		delete(r.participants, identity)
 		delete(r.participantOpts, identity)
 	}
+	if !p.Hidden() {
```

Comment:
this should be moved within the `if ok {` block. the tests are failing due to p being nil.

Reply:
thanks!